DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -481,28 +518,11 @@ public IEnumerable<Course> GetCoursesEverUsedAtCentreByCategory(int centreId, in
             );
         }
 
-        public bool DoesCourseExistAtCentre(int customisationId, int centreId, int? categoryId)
-        {

```

Comment:
Noticed this method was unused so I removed it. It's from another ticket I worked on when I made a new version of it and forgot to remove the old.